File: python/src/tests/t_buffer_max_size.py
#!/usr/bin/python3
#
# $Id:$
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

r = atrshmlog.attach()

size = atrshmlog.get_buffer_max_size()

print('buffer max size : ' + str(size) + ' : ')


print (' ')

exit(0);

# end of test

File: perl/src/tests/t_buffer_max_size.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$size = Atrshmlog::get_buffer_max_size();

print "buffer max size  : $size : \n";


print " \n";

exit (0);

# end of main
